modules/udf/README.md

```diff
@@ -16,11 +16,45 @@ module "dataset" {
   location                    = "US"
 }
 
+local {
+  parse_url_udf_ddl = <<EOT
+  CREATE FUNCTION IF NOT EXISTS parse_url(url STRING, part STRING)
+  AS (
+    CASE
+      -- Return HOST part of the URL.
+      WHEN part = 'HOST' THEN SPLIT(\`${PROJECT_ID}\`.${dataset_name}.check_protocol(url), '/')[OFFSET(2)]
+      WHEN part = 'REF' THEN
+        IF(REGEXP_CONTAINS(url, '#'), SPLIT(\`${PROJECT_ID}\`.${dataset_name}.check_protocol
+        (url), '#')[OFFSET(1)], NULL)
+      WHEN part = 'PROTOCOL' THEN RTRIM(REGEXP_EXTRACT(url, '^[a-zA-Z]+://'), '://')
+      ELSE ''
+    END
+  );"
```

Comment:
What is this quote terminating?

Reply:
great catch it was remnant from when this was in a shell script.